### PR TITLE
Table specimen

### PR DESCRIPTION
**Pull request by bldng**
Proposal targeting #115.

### CODE REVIEW

File: docs/specimens/table.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+```
+````
+
+### Sort and filter the columns
+
+If there is a need to remove a column or change their order, you can pass the optional `columns` array.
+
+```table
+span: 3
+columns: [Value, ID]
```
Comment (herrstucki) [inline docs/specimens/table.md]: Nit: I think this is nicer

```
columns:
  - Value
  - ID
```

**Comment by herrstucki** [comment]
This is a really cool and useful addition! I think it's generally good to go, but if you find time, maybe you could address the following (small) issues.

- [ ] You could be more strict with the table propTypes. E.g. `arrayOf(PropTypes.object)`
- [ ] Turn `Cell` into a component which returns `<td>`s instead of wrapping it in `<td>`s itself
- [ ] I think the cell padding is too big. I'd reduce it, so content is a bit more dense
- [ ] Please add a note that you can use Markdown syntax *inside* the table cells, and add an example
- [ ] Aren't the example tables a tad boring? They could use some 🍎 🍌 🍐 😀 
- [ ] Somehow the spanned tables don't respect the gutter between layout columns (see screenshot; there should be a space between the table and the code snippet). Is the `section` style really needed?

bildschirmfoto 2016-06-23 um 22 18 25

Maybe you can also get the "real" Markdown tables working (not a priority though) – otherwise please remove the text which suggests you can 😁

**Comment by bldng** [comment]
Thanks for the feedback!
I removed the mention of the Markdown tables for now, as only the most basic stuff is working at the moment. Rather than fixing that, I would like to add more configurability to the `columns` notation at some point. Something like:
```
columns:
  - Amount
    - format: $,.2f
    - align: right
```

**Comment by naehrstoff** [comment]
I would rather not have extensive configurability on table cells like you proposed because that adds much complexity, is difficult to make discoverable, and most likely it will not cover the very specific need you'll have by a small margin, so you'll have to find another solution anyway. But maybe I'm wrong here and the all needs can be easily fulfilled.

An alternative solution I would be interested in seeing tried out is to be able to provide a cell renderer. It should just be a regular React Component that you can register somehow.

**Pull request merged**